waveorder/io/writer.py

```diff
@@ -29,7 +23,8 @@ class WaveorderWriter:
     def __init__(self,
                  save_dir: str = None,
                  datatype: str = None,
-                 alt_name: str = None):
+                 alt_name: str = None,
+                 silence: bool = False):
```

Comment:
We could, but I find loggers typically start to reveal unwanted outputs that are present in the dependency functions so I have been avoiding them.  This was basically implemented so that during conversion it doesn't interrupt the progress as its switching positions.

Reply:
ok, no worries.  I think silence is fine for now.  Could also call it `verbose`